web/controllers/admin_controller.ex

```diff
@@ -319,6 +319,7 @@ defmodule ExAdmin.AdminController do
   end
 
   defp to_integer(:string, string), do: string
+  defp to_integer(:id, string), do: string
```

Comment:
I'm wondering if we should be attempting to convert the id to an integer? This should be tested with with both integer keys and binary keys (UUID).